### PR TITLE
Remove Phone Number field from Profile Edit form

### LINKED ISSUE #316: Remove phone number from account details
**To Reproduce**
1. Log in using the test user
2. Go to "Moje konto"

**Current behavior**
There is a form with a phone number included
image

**Expected behavior**
There is a form without a phone number included

### PR DESCRIPTION
**Pull request by darek-grund**
## Description
This PR resolves #316 

## Where should the reviewer start?
When logged in, go to **/Manage/Index** to see Profile Edit form

## Motivation and context
See #316 

## How has this been tested?
Manual tests

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by WTobor** [comment]
it looks good, but we should also remove code setting Phone Number (example from `IndexViewModel` and `ManageController` and any other place)
@darek-grund could you do it? If not, let me know ;)

**Comment by darek-grund** [comment]
@WTobor - I am unable to remove the functionality from models and conrollers so someone else would need to help with that.

What I did with in this pull request is remove `Phone Number` from template and made sure that the form still works (i.e. user is able to save the form when `Phone Number` is not present).

**Comment by WTobor** [comment]
@Mejtika could you please add changes described above? It should be a small quick change

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/ManageService.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/ManageService.cs:1]: why do we need that additional using?

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Interfaces/IManageService.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Interfaces/IManageService.cs:1]: why do we need that additional using?

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Manage/Index.cshtml [hunk 3]
```diff
@@ -42,9 +42,9 @@
             </div>
 
             @*<div class="form-group form-toogle">
-                <label asp-for="PushNotificationsEnabled"></label>
-                <input asp-for="PushNotificationsEnabled" data-toggle="toggle" data-onstyle="success" data-offstyle="danger" data-on="tak" data-off="nie" />
-            </div>*@
+                    <label asp-for="PushNotificationsEnabled"></label>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Manage/Index.cshtml:45]: why do we have changes here?

**Pull request merged**